### DISCUSSION TITLE
Edit field in table view itself instead of going to edit page.

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#12992] [Archive]
Example 
1. Change the status field in a single click
2. Edit value of any field from table
3. Enable/Disable which field can be edited from table view in collection setting itself
      - Example : Edit field in table view (Enabled/Disabled)

![unknown (1)](https://user-images.githubusercontent.com/36895131/165103661-a726ea7d-ad9e-4913-89d3-74ecc2c0829b.png)

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/12976, discussed in https://github.com/directus/directus/discussions/3874 🙂
